src/app/(home)/page.tsx

```diff
@@ -7,193 +7,204 @@ import MentorCard from "@/src/components/HomePageComponents/MentorCard/MentorCar
+							</CarouselItem>
+
+						</CarouselContent>
+					</Carousel>
+				</div>
+			</div>
+
+			<div className='bg-hero-gradient'>
+				<Container>
+					<div className="section">
+						<h1 className="section-title text-center">Helping the world one mentor at a time</h1>
+						<div className="section-text text-center">
+							<p className='pb-4'>sifting through the overwhelming content on the internet will slow you down.break through the <br /> noise abd get specific advice directly from the experts.</p>
+							<Button variant={'outline'}>Let's Find a Mentor</Button>
+						</div>
+					</div>
+				</Container>
+
+			</div>
+
+			<Container>
+				<div className="section">
+					<div className="section-content">
+						<StatCardList />
+					</div>
+				</div>
+			</Container>
+
+			<Container>
+				<div className="section">
+					<h1 className="section-title text-center ">In Demand Skills</h1>
+					<div className="section-content">
+
+						<div className="section-content-wrapper justify-around md:justify-between">
+							<SkillTag imgURL="/images/home/next.png" imgALT="next" title="react" />
+							<SkillTag imgURL="/images/home/node.png" imgALT="Nodejs" title="Nodejs" />
+							<SkillTag imgURL="/images/home/graphic.png" imgALT="graphic" title="Graphic Design" />
+						</div>
+
+						<div className="section-content-wrapper justify-around">
+							<SkillTag imgURL="/images/home/java.jpg" imgALT="javascript" title="JavaScript" />
+							<SkillTag imgURL="/images/home/ux.svg" imgALT="ux" title="Ux Design" />
+						</div>
+
+						<div className="section-content-wrapper justify-around md:justify-between ">
+							<SkillTag imgURL="/images/home/next.png" imgALT="next" title="react" />
+							<SkillTag imgURL="/images/home/node.png" imgALT="Nodejs" title="Nodejs" />
+							<SkillTag imgURL="/images/home/graphic.png" imgALT="graphic" title="Graphic Design" />
+						</div>
+
+					</div>
+
+				</div>
 
-          </div>
+			</Container>
+
+			<Container>
+				<div className="section">
+
+					<div className="section-title text-center">
+						<h1>Upcoming Sessions</h1>
+					</div>
+					<div className="section-text text-center">
+						<p>Sign Up to one of our session and start your journey</p>
+					</div>
+
+
+
+					<div className="section-card">
+						<SessionCard imgURL='/images/home/session-image1.jpg' title='Everthing Design' />
+
+						<SessionCard imgURL='/images/home/session-image2.jpg' title='Coding Newbies' />
+
+						<SessionCard imgURL='/images/home/session-image3.jpg' title='Project planning' />
+
+						<SessionCard imgURL='/images/home/session-image4.jpg' title='Quality Assurence' />
+					</div>
+					<div className='section-button'>
+						<Button variant={'secondary'}>Learn more</Button>
+					</div>
+				</div>
 
-        </div>
+			</Container>
+
+			<Container>
+				<div className="section">
+					<h1 className='section-title text-center'>Our Mission</h1>
```

Comment:
please use only one h1 tag in one page 
use section heading as h2 and text with p tag